app/jobs/csv_export_job.rb

```diff
@@ -34,9 +34,8 @@ def deploy_csv_export(csv_export)
     filename = csv_export.path_file
     filter = csv_export.filters
 
-    @deploys = Deploy.joins(
-      'INNER JOIN stages ON stages.id = deploys.stage_id INNER JOIN jobs ON jobs.id = deploys.job_id'
-    ).unscope(where: :deleted_at).where(filter)
+    @deploys = Deploy.includes(:job, {stage_with_deleted: :project}).joins(:job, { stage_with_deleted: :project })
```

Comment:
Should this be done with select to grab only what's needed from the db to reduce amount of data in memory?